homeassistant/components/climate/zwave.py

```diff
@@ -63,6 +63,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     node = zwave.NETWORK.nodes[discovery_info[ATTR_NODE_ID]]
     value = node.values[discovery_info[ATTR_VALUE_ID]]
     value.set_change_verified(False)
+    if value.index != 1:  # Only add 1 device
+        return
```

Comment:
I had to revert 80a82ef7ce1245e8bf9e42c6f2e567e4f2d95f9d to restore functionality on my thermostat.